aggregator_api/src/lib.rs

```diff
@@ -0,0 +1,897 @@
+//! This crate implements the Janus aggregator API.
+
+use crate::models::{GetTaskIdsResp, PostTaskReq};
+use janus_aggregator_core::{
+    datastore::{Datastore, Error},
+    task::Task,
+    SecretBytes,
+};
+use janus_core::{hpke::generate_hpke_config_and_private_key, time::Clock};
+use janus_messages::{Duration, HpkeAeadId, HpkeKdfId, HpkeKemId, TaskId, Time};
+use models::{GetTaskMetricsResp, TaskResp};
+use opentelemetry::{
+    global::meter,
+    metrics::{Histogram, Unit},
+    Context, KeyValue,
+};
+use querystring::querify;
+use rand::{distributions::Standard, random, thread_rng, Rng};
+use ring::constant_time;
```

Comment:
We use `ring::constant_time` elsewhere in `janus`, and don't yet have a direct dependency on `subtle` for any of the Janus crates. I also don't like `subtle`'s API: ultimately the `ct_eq` operation gives back a `Choice`, which is really a `u8`-masquerading-as-a-boolean. This is handy if you want to combine these values in a constant-time way, but not as great if you just want the outcome of checking equality as a boolean.

IIRC, `libprio-rs` doesn't depend on `ring` because we have folks that want to use `libprio-rs` who can't depend on `ring`. Since Janus doesn't have that restriction, and switching wouldn't let us drop `ring`, and `ring`'s API is (IMO) better, I think we should keep using `ring` for now.